src/components/calcite-pagination/calcite-pagination.stories.js

```diff
@@ -1,21 +1,36 @@
+import { storiesOf } from "@storybook/html";
 import { withKnobs, number, select } from "@storybook/addon-knobs";
 import { darkBackground, parseReadme } from "../../../.storybook/helpers";
 import readme from "./readme.md";
 const notes = parseReadme(readme);
 
-
-export default {
-  title: "Pagination",
-  decorators: [ withKnobs ],
-  parameters: { notes }
-};
-
-export const simple = () =>
-  `<calcite-pagination
-    start="${number("Start", 1)}"
-    total="${number("Total", 3)}"
-    num="${number("Num", 1)}"
-    dir="${select("dir", ["ltr", "rtl"],"ltr")}"
-    theme="${select("Theme", ["light", "dark"] ,"light")}"
-    background-style="${select("Background Style", ["backgroundColor", "foregroundColor"] ,"foregroundColor")}">
-  </calcite-pagination>`;
+storiesOf("Pagination", module)
+.addDecorator(withKnobs)
+.add(
+  "Simple",
+  () => `
+    <calcite-pagination
+      start="${number("start", 1)}"
+      total="${number("total", 128)}"
+      num="${number("num", 20)}"
+      dir="${select("dir", ["ltr", "rtl"],"ltr")}"
+      theme="light"
+    >
+    </calcite-pagination>
+  `,
+  { notes }
+)
+.add(
+  "Dark Mode",
+  () => `
+    <calcite-pagination
+      start="${number("start", 1)}"
+      total="${number("total", 128)}"
+      num="${number("num", 20)}"
+      dir="${select("dir", ["ltr", "rtl"],"ltr")}"
+      theme="dark"
+    >
+    </calcite-pagination>
+  `,
+  { notes, backgrounds: darkBackground }
+);
```

Comment:
😈

Reply:
Oh this isn't showing up in the preview, but this file doesn't have a newline at the end of it.

Reply:
Pretty sure I have no control over that at this point. Formatter runs automatically on git add now and it's in the prettier god's hands IIRC

Reply:
I think the final newline is actually not a prettier concern, but an editor one. If you *want*, you can add this to your vscode config:

```
  "files.insertFinalNewline": true,
  "files.trimFinalNewlines": true,
```